src/components/layout/CyVerseAppBar.js

```diff
@@ -603,7 +576,7 @@ function CyverseAppBar(props) {
                 })}
             >
                 <Toolbar>
-                    <Hidden only={["sm", "md", "lg", "xl"]}>
+                    <Hidden smUp>
```

Comment:
This and similar is so we're using the same names for things throughout -- in this file, we only use `smUp` and `xsDown`, but we were writing those four ways (those two, plus the versions using the `only` prop). So, just minor consistency, but anyway.